builder/create-dockerfile/src/create_dockerfile.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+use Symfony\Component\Yaml\Yaml;
+
+class GenFiles
```

Comment:
We could probably have unit tests for this class, but the functionality is pretty simple. If for whatever reason, we want to add more feature detection (extensions perhaps) from the `app.yaml`, we'd probably not want to need end-to-end tests for each feature.